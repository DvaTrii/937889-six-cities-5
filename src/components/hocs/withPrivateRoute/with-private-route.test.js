import React from "react";
import renderer from "react-test-renderer";
import {withPrivateRoute} from "./withPrivateRoute";

const MockComponent = () => {

  return (
    <React.Fragment>
      <div></div>
    </React.Fragment>
  );
};
const authorizationStatus = true;

const MockComponentWrapped = withPrivateRoute(MockComponent, authorizationStatus);

it(`withPrivateRoute is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
