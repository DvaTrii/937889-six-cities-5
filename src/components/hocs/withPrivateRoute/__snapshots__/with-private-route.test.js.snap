// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withPrivateRoute is rendered correctly 1`] = `<div />`;
